myapp/spec/system/tasks_spec.rb

```diff
@@ -53,78 +56,104 @@
         expect(page).to have_content new_description
       end
 
-      # Failure case will be added when validation is added.
+      it 'failed to create a task due to blank title' do
+        fill_in 'title', with: ''
+        fill_in 'description', with: 'ThisIsDescription'
+        click_on 'Create'
+
+        expect(current_path).to eq tasks_path
+        expect(page).to have_content TasksController::MSG_CREATE_FAILURE
```

Comment:
You should better test the string directly  instead of `TasksController::MSG_CREATE_FAILURE`

Reply:
Thanks fro the suggestion, I addressed [it](https://github.com/Fablic/training/pull/1706/commits/138f123b42d5e2ec8d627b7c3dd52c6eb71bacc4)